### DISCUSSION TITLE
Comment on HiLetgo PCM5102 I2S IIS Lossless Digital Audio DAC Decoder

### DISCUSSION THREAD

**Discussion by PRSXFENG** [pschatzmann/arduino-audio-tools#1806] [General]
Hi there I would just like to inform on some boards of this design, the H and L pads may be flipped, I was troubleshooting why no audio was coming out of my board, until I took a multi-meter to verify connections, and found that on my board, the H and L are flipped, with H being connected to GND

As such, I had to invert the connections for the board to function as expected

Just leaving this here in case anyone also faces this problem
Possibly related to the issue in https://github.com/pschatzmann/arduino-audio-tools/discussions/1641

**Comment by alw1746** [comment]
FYI, the [Adafruit PCM5102](https://www.adafruit.com/product/6250) board, worked for me with just 3V,GND and 3 I2S pins. With the HiLetgo, I had to jumper SCK to GND and solder bridge XMT:H before it worked.

**Comment by pschatzmann** [comment]
I updated [my documentation](https://github.com/pschatzmann/arduino-audio-tools/wiki/External-DAC#hiletgo-pcm5102-i2s-iis-lossless-digital-audio-dac-decoder).
